### ISSUE TITLE
Library is misidentified: velocity-tools-2.0.jar appears to be cpe:/struts-2.0.0

### ISSUE THREAD

**Issue by ChristopherSchultz** [dependency-check/DependencyCheck#889]
False positive on library `velocity-tools-2.0.jar` - reported as `cpe:/a:apache:struts:2.0.0`

Here's the information from the dependency report:

```
Source | Name | Value
-- | -- | --
file | name | velocity-tools
file | version | 2.0
jar | package name | apache
jar | package name | tools
jar | package name | velocity
Manifest | extension-name | velocity-tools
Manifest | Implementation-Title | org.apache.velocity
Manifest | Implementation-Vendor | Apache Software Foundation
Manifest | Implementation-Vendor-Id | org.apache
Manifest | Implementation-Version | 2.0.1-SNAPSHOT
Manifest | specification-title | VelocityTools is a set of utilities for use with the Velocity template engine and Struts web framework
Manifest | specification-vendor | Apache Software Foundation

Identifiers

    cpe: cpe:/a:apache:struts:2.0.0   Confidence:HIGHEST   

```

**Comment by ChristopherSchultz** [comment]
I checked with dependency-checker 3.1.0 and `velocity-tools-2.0.jar` is still being identified as `cpe:/a:apache:struts:2.0.0`.

In the "evidence" section, the only place Struts is mentioned is in the "specification title" where the library identifies itself as utilities for "velocity and struts".

What other information can I provide to help get this resolved?

**Comment by jeremylong** [comment]
I thought I had put a suppression rule to resolve this - I apologize if I missed this one and marked it as closed (or maybe something else is going on). I will do a sweep of all fo the FP reports before the next release.

**Comment by jeremylong** [comment]
I just tested this on my system using ODC 3.0.0 through the current snapshot (3.1.3-SNAPSHOT) and none of them reported velocity-tools as struts.  Now, velocity-tools does have Struts as a direct dependency - but that is a different issue that might be able to be resolved by adding excludes to the velocity-tools dependency node in the pom.xml.

**Comment by ChristopherSchultz** [comment]
I'm sorry I didn't see this a few months ago when you commented on and closed it.

I'm not using Maven, so there aren't any hidden external dependencies being downloaded.

I have tried with 3.3.1 and velocity-tools no longer appears to be identified as struts-1.x.
